doc/api/http2.md

```diff
@@ -2059,13 +2059,17 @@ The socket timeout logic is set up on connection, so changing this
 value only affects new connections to the server, not any existing connections.
 
 ### `http2.createServer(options[, onRequestHandler])`
+
+<!--lint disable nodejs-yaml-comments -->
```

Comment:
Why disable the in this block?

Reply:
That’s because one item in the changes list has been introduced without a PR on node repo for security reason (3948830ce6408be620b09a70bf66158623022af0). This is (will be) picked up by the linter as invalid (see https://github.com/nodejs/remark-preset-lint-node/runs/1235246278?check_suite_focus=true)

Ideally the linter should be aware of that pattern, but I figured disabling linter on the few comments it happens would be good enough on a first iteration.

Reply:
Is the pattern that if the `pr-url` is for `nodejs-private/node-private`, then it should have a matching `commit` on frontmatter?

Reply:
OK, I agree that’s a cleaner solution. I didn’t take the time to implement that on the linter PR, I’ll try to do that later today.

Reply:
No worries, just talking the problem out loud 😄 
1. Should the `commit` be a URL as well?
1. Should the `commit` be disallowed on non-private `pr-url`s?

Reply:
> Should the commit be a URL as well?

Before I started working on refactoring YAML comments, there were 14 occurrences of commit IDs:

* 13 were `commit: sha`
* 1 was `pr-url: https://github.com/nodejs/node/commit/sha`

I'd stick with the majority here, as we did on the previous PR, unless someone objects.

> Should the commit be disallowed on non-private pr-urls?

Yes, the goal is consistency, let's avoid it where we can. My plan is to have it allowed only on changes introduced on Node.js 0.x (because sometimes changes were made directly on master), and on security releases.